src/ble.rs

```diff
@@ -52,7 +52,12 @@ pub async fn ble_advertise_task(
     );
     notifying_characteristic.lock().set_value(b"uptime: 0");
 
-    advertising.name(name).add_service_uuid(*UUID_BLE_SERVICE);
+    // advertising.name(name).add_service_uuid(*UUID_BLE_SERVICE);
+    advertising.set_data(
+        BLEAdvertisementData::new()
+          .name("ESP32-GATT-Server")
+          .add_service_uuid(BleUuid::Uuid16(0xABCD)),
```

Comment:
this is not really how it was before lol

Reply:
My apologies, I was copying from the example, and forgot to clean it up.